client/src/components/layout/EditForm.js

```diff
@@ -0,0 +1,190 @@
+import React, { useState, useEffect } from "react"
+import { Redirect } from "react-router-dom"
+
+import translateServerErrors from "../../services/translateServerErrors.js"
+import ErrorList from "./ErrorList.js"
+
+const EditForm = (props) => {
+  const [form, setForm] = useState({
+    title: "",
+    continent: "",
+    country: "",
+    city: "",
+    numberOfDays: "",
+    description: "",
+    userId: "",
+    id: ""
+  })
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+  const [errors, setErrors] = useState([])
+
+  const tripId = props.match.params.id
+
+  const findTrip = async () => {
```

Comment:
prefer `getTrip` over `findTrip` for consistency. This is only a suggestion though